hoot-core/src/main/cpp/hoot/core/util/StringUtils.h

```diff
@@ -65,6 +69,14 @@ class StringUtils
    * @return true if the input has at least one alphabetic character; false otherwise
    */
   static bool hasAlphabeticCharacter(const QString input);
+
+  /**
+   * Converts a JSON string to a Boost property tree
+   *
+   * @param jsonStr the string to convert
+   * @return a Boost property tree
+   */
+  static boost::shared_ptr<boost::property_tree::ptree> jsonStringToPropTree(QString jsonStr);
```

Comment:
This is good.  We use this in a few different places that can all just call this function.